apps/src/applab/AppLabCrosshairOverlay.jsx

```diff
@@ -0,0 +1,40 @@
+/** @file App Lab-specific Crosshair Overlay */
+
+import CrosshairOverlay from '../templates/CrosshairOverlay';
+import { scaledDropPoint } from './gridUtils';
+
+const AppLabTooltipOverlay = React.createClass({
+  propTypes: {
+    // width, height, mouseX and mouseY are given in app-space, not screen-space
+    width: React.PropTypes.number,
+    height: React.PropTypes.number,
+    mouseX: React.PropTypes.number,
+    mouseY: React.PropTypes.number
+  },
+
+  render() {
+    var mouseX = this.props.mouseX;
+    var mouseY = this.props.mouseY;
+
+    // Modify passed props if we're in a 'dragging' mode.
+    var draggingElement = $(".ui-draggable-dragging");
+    this.isDragging_ = !!draggingElement.length;
+    if (this.isDragging_) {
```

Comment:
Ah, yep.  It was used somewhere else at some point but got refactored into a small enough module that it should be local.  Thanks for catching this!